src/styles/main.css

```diff
@@ -20,6 +25,131 @@
+.form-control:not([type="checkbox"]):read-only,
+.form-control:not([type="radio"]):read-only {
+  box-shadow: inset 0 0 0 1px var(--KInputBorder, var(--grey-300)) !important;
+}
+
+.k-input:not([type="checkbox"]),
+.swagger-ui .operations-container select:not([type="checkbox"]),
+.swagger-ui .parameters input[type="email"]:not([type="checkbox"]),
+.swagger-ui .parameters input[type="password"]:not([type="checkbox"]),
+.swagger-ui .parameters input[type="search"]:not([type="checkbox"]),
+.swagger-ui .parameters input[type="text"]:not([type="checkbox"]),
+.swagger-ui .parameters textarea:not([type="checkbox"]),
+.swagger-ui .parameters select:not([type="checkbox"]),
+.swagger-ui .opblock .content-type-wrapper select:not([type="checkbox"]),
+.swagger-ui
+  .opblock
+  .responses-wrapper
+  .opblock-section-header
+  select:not([type="checkbox"]),
+.swagger-ui .scheme-container .actions select:not([type="checkbox"]),
+.swagger-ui .scheme-container .version-select select:not([type="checkbox"]),
+.k-input:not([type="radio"]),
+.swagger-ui .operations-container select:not([type="radio"]),
+.swagger-ui .parameters input[type="email"]:not([type="radio"]),
+.swagger-ui .parameters input[type="password"]:not([type="radio"]),
+.swagger-ui .parameters input[type="search"]:not([type="radio"]),
+.swagger-ui .parameters input[type="text"]:not([type="radio"]),
+.swagger-ui .parameters textarea:not([type="radio"]),
+.swagger-ui .parameters select:not([type="radio"]),
+.swagger-ui .opblock .content-type-wrapper select:not([type="radio"]),
+.swagger-ui .opblock-body .examples-select select ,
+.swagger-ui
+  .opblock
+  .responses-wrapper
+  .opblock-section-header
+  select:not([type="radio"]),
+.swagger-ui .scheme-container .schemes select,
+.swagger-ui .scheme-container .version-select select:not([type="radio"]),
+.form-control:not([type="checkbox"]),
+.form-control:not([type="radio"]) {
+  display: block;
+  padding: 10px var(--spacing-md, 16px);
+  color: var(--KInputColor, var(--black-70, rgba(0, 0, 0, 0.7)));
+  font-size: var(--type-sm, 14px);
+  font-weight: normal;
+  line-height: 24px;
+  border: none;
+  border-top-color: initial;
+  border-top-style: none;
+  border-top-width: initial;
+  border-right-color: initial;
+  border-right-style: none;
+  border-right-width: initial;
+  border-bottom-color: initial;
+  border-bottom-style: none;
+  border-bottom-width: initial;
+  border-left-color: initial;
+  border-left-style: none;
+  border-left-width: initial;
+  border-image-source: initial;
+  border-image-slice: initial;
+  border-image-width: initial;
+  border-image-outset: initial;
+  border-image-repeat: initial;
+  border-radius: 3px;
+  border-top-left-radius: 3px;
+  border-top-right-radius: 3px;
+  border-bottom-right-radius: 3px;
+  border-bottom-left-radius: 3px;
+  background-color: var(--KInputBackground, var(--white, #ffffff));
+  box-shadow: inset 0 0 0 1px var(--KInputBorder, var(--grey-300)) !important;
```

Comment:
Please do no use and overwrite kongponents variables

If needed - please create a new variable and set value to corresponding value